source/game.c

```diff
@@ -128,12 +126,11 @@
 
 #define NEXT_ROUND_BTN_SEL_X 0
 
-#define GAME_PLAYING_HAND_SEL_Y        0
-#define GAME_PLAYING_BUTTONS_SEL_Y     1
-#define GAME_PLAYING_PLAY_BTN_SEL_X    0
-#define GAME_PLAYING_DISCARD_BTN_SEL_X 1
-#define GAME_PLAYING_NUM_SEL_ROWS      2
-#define GAME_PLAYING_NUM_BOTTOM_BTNS   2
+#define GAME_PLAYING_HAND_SEL_Y    1
+#define GAME_PLAYING_BUTTONS_SEL_Y 2
+
+#define GAME_PLAYING_NUM_SEL_ROWS    2
+#define GAME_PLAYING_NUM_BOTTOM_BTNS 2
```

Comment:
```suggestion
#define GAME_PLAYING_HAND_SEL_Y      1
#define GAME_PLAYING_BUTTONS_SEL_Y   2
#define GAME_PLAYING_NUM_SEL_ROWS    2
#define GAME_PLAYING_NUM_BOTTOM_BTNS 2
```

Maybe remove the space between the defines since they're related.